### ISSUE TITLE
VoiceOver users cannot jump to the quoted message

### ISSUE THREAD

**Issue by MarcoZehe** [signalapp/Signal-iOS#6079]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
VoiceOver users cannot jump to an original message if the current message is a reply to said original. Sighted users can just tap on the quote and jump directly to the original message.

### Steps to reproduce
- Open a chat that has multiple messages, and at least one is a reply to another message.
- Turn on VoiceOver
- Focus the reply
- Try to jump to the original. Double-tapping doesn't work, and there is no action in the action rotor to jump to the original message, either. It won't work.

**Actual result:** Nothing happens, current message stays in VoiceOver focus, and the screen doesn't scroll

**Expected result:** Choosing an appropriate action, VoiceOver users should be able to jump directly to the original message. Screen should scroll, and VoiceOver focus should be placed on the message item for the original message.

### Device info

**Device**: iPhone 16 Pro max

**iOS version**: 26 Beta 2, but also happens in 18 (any release)

**Signal version:** Any current or recent version, this has never worked

**Comment by DO9RE** [comment]
> 
> 
> - [x] I have searched open and closed issues for duplicates
> - [x] I am submitting a bug report for existing functionality that does not work as intended
> - [x] This isn't a feature request or a discussion topic
> 
> ----------------------------------------
> 
> ### Bug description
> VoiceOver users cannot jump to an original message if the current message is a reply to said original. Sighted users can just tap on the quote and jump directly to the original message.
> 
> ### Steps to reproduce
> - Open a chat that has multiple messages, and at least one is a reply to another message.
> - Turn on VoiceOver
> - Focus the reply
> - Try to jump to the original. Double-tapping doesn't work, and there is no action in the action rotor to jump to the original message, either. It won't work.
> 
> **Actual result:** Nothing happens, current message stays in VoiceOver focus, and the screen doesn't scroll
> 
> **Expected result:** Choosing an appropriate action, VoiceOver users should be able to jump directly to the original message. Screen should scroll, and VoiceOver focus should be placed on the message item for the original message.
> 
> ### Screenshots
> 
> 
> 
> ### Device info
> 
> **Device**: iPhone 16 Pro max
> 
> **iOS version**: 26 Beta 2, but also happens in 18 (any release)
> 
> **Signal version:** Any current or recent version, this has never worked
> 
> ### Link to debug log
> 
> 
> 
> 

I'm blind myself, using an iPhone 13 with latest iOS at this moment. Can verify the described issue.

**Comment by MarlowBrown** [comment]
Hey @MarcoZehe,

I noticed you have another VoiceOver ticket, and I'll look into both when I have a chance. As always, thank you for posting!

**Comment by MarlowBrown** [comment]
Keep open

**Comment by DO9RE** [comment]
I am a blind Voiceover User on iOs and would like to foster the topic as well.

**Comment by Simon818** [comment]
Necessary context (in my opinion) is that VoiceOver also doesn't read the reply preview above the reply itself, so it is actually impossible for us VoiceOver users to read the original message at all.

The entire reason replies exist is to eliminate the need to add necessary context to a message. But then the app effectively locks us out of gaining that context.

Some strategies for solving this are discussed in #5707. That original ssue is not entirely accurate now--Voiceover *does* tell me when a message is a reply, and it names the person being replied to, but I still can't read the quoted message.
